lib/rules/at-rule-descriptor-value-no-invalid/index.cjs

```diff
@@ -0,0 +1,68 @@
+// NOTICE: This file is generated by Rollup. To modify it,
+// please instead edit the ESM counterpart and rebuild with Rollup (npm run build).
+'use strict';
+
+const cssTree = require('css-tree');
+const nodeFieldIndices = require('../../utils/nodeFieldIndices.cjs');
+const atKeywords = require('../../reference/atKeywords.cjs');
+const report = require('../../utils/report.cjs');
+const ruleMessages = require('../../utils/ruleMessages.cjs');
+const validateOptions = require('../../utils/validateOptions.cjs');
+
+const UNSUPPORTED_NESTING_REGEX = new RegExp(
+	`^((?!${[...atKeywords.nestingSupportedAtKeywords.values()].join('|')}).)*$`,
+	'i',
+);
```

Comment:
I'd like to unify it later.
See: https://github.com/stylelint/stylelint/pull/8197/files#diff-709b8ceefdd8d9a3b4b446f29f245ce253e36a5cc9843d695c7bfcbadb78b8d6R8-R11